cli/flags.go

```diff
@@ -145,6 +141,18 @@ duration.`),
 Adjusts the max idle time of the scanner. This speeds up the scanner on small
 clusters to be more responsive.`),
 
+	"ssl-ca": wrapText(`
+Path to the CA certificate. Needed by servers and clients in secure mode.`),
```

Comment:
s/servers and clients/clients and servers/  (for consistency with how you phrase them elsewhere).